nana/modules/database/lang_db.py

```diff
@@ -0,0 +1,34 @@
+from sqlalchemy import Column, String, UnicodeText
+import threading
+from nana import BASE, SESSION
+
+
+class Locales(BASE):
+    __tablename__ = "locales"
+    chat_id = Column(String(14), primary_key=True)
+    locale_name = Column(UnicodeText)
+
+    def __init__(self, chat_id, locale_name):
```

Comment:
Codacy found an issue: [Missing docstring in __init__ (D107)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330175)